packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -110,6 +110,61 @@ function ListView({
   const fetchClient = useFetchClient();
   const { post, del } = fetchClient;
 
+  const entityListQuery = useQuery(
```

Comment:
Same here, i think this out to be a different PR, there's probably a lot more optimisations / better practice we could be doing in a focussed situation.